src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/PortalSpawnModifier.java

```diff
@@ -55,11 +58,41 @@ public void portalSpawn(CreatureSpawnEvent event) {
 				event.setCancelled(true);
 				return;
 			}
-			if (roll(ghastChance)) {
+
+			//Disabled since we don't want ghasts spawning in nether portals.
```

Comment:
Don't comment out code blocks, remove it if you're not gonna use it

Reply:
Also why would we not want ghasts from nether portals, that should be the primary/only way to properly farm them, no? And even if we were to not want it, hide it behind a config option instead of removing the code

Reply:
fixed with the updated edit.